src/modules/menu/components/SimpleCategory.tsx

```diff
@@ -7,6 +7,7 @@ const SimpleCategory: React.FC<CategoryProps> = ({ category, foods }) => {
     <List
       style={{
         scrollMarginTop: "112px", // very important for anchor to work
+        overflow: "hidden",
```

Comment:
Remove this line it make transition when click food category on topnav (look weird)

Reply:
Done